Dockerfile

```diff
@@ -1,15 +1,9 @@
-FROM golang:alpine
-
-RUN apk add --update --no-cache git
-RUN go get github.com/honeycombio/honeyaws/cmd/honeyelb
-RUN go get github.com/honeycombio/honeyaws/cmd/honeyalb
-RUN go get github.com/honeycombio/honeyaws/cmd/honeycloudfront
-RUN go get github.com/honeycombio/honeyaws/cmd/honeycloudtrail
-
-FROM alpine
+FROM golang:1.9-alpine
+COPY . /go/src/github.com/honeycombio/honeyaws
+WORKDIR /go/src/github.com/honeycombio/honeyaws
+RUN go install ./...
 
+FROM golang:1.9-alpine
 RUN apk add --update --no-cache ca-certificates
 COPY --from=0 /go/bin/honeyelb /usr/bin/honeyelb
 COPY --from=0 /go/bin/honeyalb /usr/bin/honeyalb
-COPY --from=0 /go/bin/honeycloudfront /usr/bin/honeycloudfront
-COPY --from=0 /go/bin/honeycloudtrail /usr/bin/honeycloudtrail
```

Comment:
why remove honeycloudfront and honeycloudtrail?
Also, we _are_ using honeycloudfront, if you remove this it will break that deployment.

Reply:
(I think you touched this file only for testing and forgot to remove the changes?)